pkgs/tools/networking/kaput-cli/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, mkYarnPackage, fetchFromGitHub }:
+
+mkYarnPackage rec {
```

Comment:
This should be probably converted to be in nodePackages.

Reply:
I am having a hard time tracking down some sort of guide to get this done. I checked https://nixos.org/manual/nixpkgs/stable/#node.js which pointed me to https://github.com/svanderburg/node2nix, but I cannot find any explicit information on how to prepare a `default.nix` for nixpkgs.

I've also tried reading through some existing packages that make use of `nodePackages` but could not figure things out.

Do you have anything to suggest?

Reply:
I've added the npm package to `pkgs/development/node-packages/node-packages.json`. I hope this was what you meant initially.

It builds and works find as `nodePackages.kaput-cli` but it does not build on latest (`nodePackages_latest.kaput-cli`).

Is this ok to leave as is, or is there a way to mark the broken one as such?

Reply:
> I've added the npm package to pkgs/development/node-packages/node-packages.json. I hope this was what you meant initially.

Yes

> It builds and works find as nodePackages.kaput-cli but it does not build on latest (nodePackages_latest.kaput-cli).

The default is nodePackages and I usually don't worry about nodePackages_latest.